keyring/tests/test_backend.py

```diff
@@ -14,7 +14,9 @@
 from .util import random_string
 from keyring import errors
 
-DIFFICULT_CHARS = string.whitespace + string.punctuation
```

Comment:
Sure, you're right. I just went the easiest path.

But I'm not sure what do you mean by "even better would be to find a way to encode these characters such that they're not given special treatment w.r.t. other backends".

I thought about overwriting that property only for migration test, would it be ok?